addons/l10n_gcc_pos/static/src/xml/OrderReceipt.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension" owl="1">
+
+        <xpath expr="//div[hasclass('pos-receipt-contact')]" position="after">
+            <t t-if="receipt.is_gcc_country">
+                <br/>
+                <br/>
+                <div class="pos-receipt-header">
+                    <span id="title_english">Tax Invoice</span>
+                </div>
+                <div class="pos-receipt-header">
+                    <span id="title_arabic">الفاتورة الضريبية</span>
```

Comment:
@mart-e isn't this going to have odd effects with translations? Does the JS API support ways to disable translations and ensure these remain as-is?

Reply:
the best solution should be to have module that allow to use an extra language than the one set on the user preference and using the translation to display both languages. Because here, if people translate this module this will break the display in english and arabic.

BTY, because this module, specific to a localisation, should be merged before beginning of December (in stable) because it is enforced by the law of this country, making a generic way to handle multiple languages was not really possible. What we should  probably do is having a module that gives the possibilities to display multiple language on the receipt in master.

Reply:
> Because here, if people translate this module this will break the display in english and arabic.

Well yes that's pretty much the point, and nothing will stop them from translating the module so it will definitely happen. So the translations inside the view needs to be disabled.

I know this is feasible in qweb/py using `t-translation="off"`, but I don't know if it works in qweb/js or owl or whatever this thing is. If it does it should definitely be used.

Reply:
@xmo-odoo Thanks, I'll ask to the framework JS

Reply:
@bakboka The JS team told me that the attribute translation="off" should work in the qweb JS. So you can use it to avoid issues of translation.

But to correcty use it you'll also need to apply the comment with the "t-if" and remove the replace. Otherwise you'll break the multi-company stuff.

Reply:
We can also just keep it out of the .pot

Reply:
it won't change the fact that we need to change the xpath replace. So if we do so, it is probably easier to add "translate="off"" than making an esception to not generate .pot for this module

Reply:
We don't publish translations on transifex for localisation modules but `t-translation="off"` makes sense here because you still want to prevent that a user translate the English part in Arabic (and reverse)